net/nimble/host/services/bas/syscfg.yml

```diff
@@ -0,0 +1,32 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+syscfg.defs:
+    BLE_SVC_BAS_BATTERY_LEVEL_PERM:
+        description: >
+            Defines permissions for writing "Battery Level" characteristics. Can
+            be zero to allow write without extra permissions or combination of:
+                BLE_GATT_CHR_F_READ_ENC
+                BLE_GATT_CHR_F_READ_AUTHEN
+                BLE_GATT_CHR_F_READ_AUTHOR
+        value: 0
+    BLE_SVC_BAS_BATTERY_LEVEL_NOTIFY:
+        description: >
+            Set to BLE_GATT_CHR_F_NOTIFY to support notification,
+            or 0 to disable it.
+        value: BLE_GATT_CHR_F_NOTIFY
```

Comment:
this should be a simple 0/1 value - it's not a permissions mask, just simple bool to either enable or disable notify prop

Reply:
or I suppose the spec allows to permission these? So you could make it a permissions mask?

Reply:
Based on some informations scattered in different places of spec, notifications should most likely follow permissions of CCCD. However, NimBLE does not allow to set them and obviously also does not enforce them so there's nothing to set here actually :)

I'd leave this as NOTIFY_ENABLE 0/1 syscfg for now and I'll look into adding appropriate support in Mynewt - we can then add NOTIFY_PERM syscfg and update code accordingly.